robotoff/app/api.py

```diff
@@ -1141,6 +1142,31 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         resp.media = response
 
 
+class UnansweredQuestionCollection:
+    def on_get(self, req: falcon.Request, resp: falcon.Response):
+        response: JSONType = {}
+        page: int = req.get_param_as_int("page", min_value=1, default=1)
+        count: int = req.get_param_as_int("count", min_value=1, default=25)
+        question_type: str = req.get_param("question_type")
+
+        insights = list(
+            get_insights(
+                keep_types=[question_type],
+                group_by=True,
+                annotation=0,
```

Comment:
No what we really want are unanswered questions, that is with annotation=None.

See the documentation of this attribute in models.py:

```python

    # The annotation of the given insight. Four possible values are possible:
    # null = This insight has not been annotated
    #  -1 = Rejected
    # 0 = 'I don't know'
    # 1 = Validated
    annotation = peewee.IntegerField(null=True, index=True)
```